config/modules/defaults.py

```
@@ -0,0 +1,163 @@
+    """
+    return key.endswith(('_COOLDOWN_MINUTES', '_COOLDOWN_SECONDS'))
+
+def get_cooldown_keys() -> List[str]:
+    """
+    Get a list of all cooldown-related configuration keys.
+    
+    Returns:
+        List of cooldown configuration keys
+    """
+    defaults = create_default_config()
+    return [key for key in defaults.keys() if is_cooldown_key(key)]
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)